Unix/http/httpcommon.c

```diff
@@ -214,6 +214,12 @@ MI_Result CreateSSLContext(SSL_CTX **sslContext, SSL_Options sslOptions)
         options |= SSL_OP_NO_TLSv1_2;
     }
 #endif
+#ifdef SSL_OP_NO_COMPRESSION
```

Comment:
What sets `SSL_OP_NO_COMPRESSION`? I don't see anything in this checkin that does that.

Here, you do an `#ifdef`, but you don't do that in other places (cli_c.c, options.c, server.c). I'd like to see consistency; either have an `#ifdef`, and be consistent, or don't have an `#ifdef` anywhere.

In general, if there's never a reason to disable an option, I'm not a fan of `#ifdef`, it just increases complexity and testing matrix. Is there a reason to disable this? If not, please get rid of it. If so, then add `#ifdef` tests to other usages.